src/plugin.rs

```diff
@@ -52,6 +56,9 @@ macro_rules! appbase_plugin_default {
       fn state(&self) -> PluginState {
          self.base.state
       }
+      fn handle(&mut self) -> Option<JoinHandle<()>> {
+         std::mem::replace(&mut self.base.handle, None)
```

Comment:
This can be replaced by the next:
```suggestion
         self.base.handle.take()
```